### ISSUE TITLE
Version used

### ISSUE THREAD

**Issue by Smandal-physna** [nejlag/Semi-Supervised-Learning-GAN#5]
Hello
Can you please which version of tensorflow and keras you have used to run this code on
Thanks and great work !

**Comment by nejlag** [comment]
Hi, 
Unfortunately, I didn't document the version. But, given the fact that I wrote this in late 2017, I think tensorflow 1.4.0 should work! It's pure tensorflow. You don't need keras to run this.

**Comment by Smandal-physna** [comment]
Great thanks. I was trying on tensorflow v2 and it seems to work on v1.
Another followup question, according to the paper, the code should run on less number of images.. So after you take as input the whole mnist dataset, can you point out where you choose a low subset of images to train the GAN architecture on? or do you train it on the whole mnist data? (Again I really appreciate this I know how difficult it is to go back to old codes and provide explanations)

**Comment by nejlag** [comment]
Should run on **less number of labeled images** not less images. _labeled_rate_ controls the percentage of labeled images per batch. _get_labled_mask(...)_ provides a mask to filter images of each batch (which image should have label and which shouldn't).

**Comment by Smandal-physna** [comment]
Great thank you for the help!
